frontend/src/pages/LoginPage.js

```diff
@@ -14,37 +16,89 @@ const useStyles = makeStyles(theme => ({
 		boxSizing: "border-box"
 	},
 	form: {
-		backgroundColor: "#56CCF2",
+		backgroundColor: theme.palette.primary.main,
+		opacity:0.8,
+		width: '100vw',
+        height: '100vh'
+
 	},
 	svg: {
-		position: 'absolute',
```

Comment:
Image should be horizontally aligned with the text and have more space in between it.

Reply:
```
	svg: {
		position:"absolute",
		bottom: 0,
		marginBottom: "2rem",
		left: 0,
		marginLeft: "6rem",
		[theme.breakpoints.down('md')]:
		{
			height: 250,
			bottom: 250,
			position:"static",
		},
		[theme.breakpoints.down(820)]:
		{
			marginBottom: "-4rem",
			marginLeft: "-2rem",
			bottom: 250,
			position:"static",
		},
```

I think this is a best compromise since in small screens it is not desirable for the image to be absolute since it can go over the title in some screens so it is statically in the back of the input fields. The scroll down effect is no longer present